src/app/core/header/header.component.ts

```diff
@@ -13,15 +13,18 @@ import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from
 })
 export class HeaderComponent {
 
+  constructor(readonly dataSourceService: DataSourceService) {
```

Comment:
Most likely there is no reason to have this injected (unless I'm wrong about the href in the html file)

Reply:
That's where the hubmapPortalUrl is injected from, but this should probably be provided via an @Input from app.component

Reply:
@Input() hubmapPortalUrl: string